src/Toolbox.php

```diff
@@ -3309,4 +3313,14 @@ final public static function getNormalizedItemtype(string $itemtype)
     {
         return strtolower(str_replace('\\', '', $itemtype));
     }
+
+    /**
+     * @param string $msg
+     * @return array|string|string[]
+     */
+    public static function cleanPaths(string $msg): string|array
```

Comment:
I guess we can just keep this one (since it's static and eays to call anywhere in GLPI), drop the other one in log line formatter (replaced with TOolbox one call), and also use this one for each sanitizing instead of hardcoded replacements in ErroHandler/ErrorController.

Reply:
You may create a `Glpi\Error\ErrorUtils` class that contains a static `cleanPaths` method to not depend on the whole `Toolbox` class.

Reply:
The point is we should onlyç have one method that does the replacement, and rely on it elsewhere. Cedric's proposal seems good to me.